### ISSUE TITLE
Tickets turnover and sold contains unpaid tickets

### ISSUE THREAD

**Issue by AEtherC0r3** [openSUSE/osem#1473]
admin/tickets#index and #show include tickets that have not been paid.

To recreate:
1. As an attendee select tickets (with non zero price) to buy
2. Press the Continue button with the little cart
3. Do not make the payment
4. Go to admin/tickets#index, the columns sold and turnover will have taken into account the unpaid tickets

**Comment by alator21** [comment]
I can work on  it.

**Comment by differentreality** [comment]
Good catch @AEtherC0r3 
@alator21 go ahead :)

**Comment by differentreality** [comment]
It also seems that in admin/tickets#index the total number of tickets (and of turnover) changes to the newest amount of tickets being ready to be purchased, ie I chose to buy 2 tickets, but then didn't. The next day I chose to buy 5 tickets (but haven't paid yet). What is shown in admin/tickets#index is the 5 tickets (latest tickets I wanted to buy, plus the latest any other user would like to buy).
